chat-android/src/main/java/com/ably/chat/Typing.kt

```diff
@@ -67,7 +51,7 @@ public interface Typing : EmitsDiscontinuities {
      * once the timer expires, a typingStopped event will be emitted. The timeout is configurable through the typingTimeoutMs parameter.
      * If the current user is already typing, it will reset the timer and being counting down again without emitting a new event.
      */
-    public suspend fun start()
+    public suspend fun keyStroke()
```

Comment:
I believe it's one word, should be camelcased
```suggestion
    public suspend fun keystroke()
```

Reply:
https://github.com/ably/ably-chat-kotlin/pull/117/commits/597efb0ab27418f753c6e99e4d6423042d3ae1ab